modules/gdscript/gdscript_byte_codegen.h

```diff
@@ -30,6 +30,7 @@
 
 #pragma once
 
+#include "core/templates/rb_map.h"
```

Comment:
Nit: Should be in a separate block of includes below the `"gdscript_*"` ones.
"Local" includes in a module should come first.